angular.json

```diff
@@ -28,6 +28,7 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
```

Comment:
This is fine for now, but will likely need to be updated later with Revature specific themes.